data/avro-schema/src/main/resources/avro/net/corda/data/flow/event/external/ExternalEventContext.avsc

```diff
@@ -0,0 +1,18 @@
+{
+  "type": "record",
+  "name": "ExternalEventContext",
```

Comment:
I think this should be baked into the external event rather than on the individual request types. We probably shouldn't be polluting client-specific data structures with our stuff. (They will need to read it, but it's part of our API, not theirs.)

Reply:
Needs to be on the underlying payloads so that each processor can receive whatever objects they want rather than one specified by the external event framework.